examples/sandbox/index.ts

```diff
@@ -1139,7 +1137,8 @@ $cosmosTx.on("click", async (e) => {
     return;
   }
   if (core.supportsCosmos(wallet)) {
-    const unsigned: core.Cosmos.StdTx = cosmosTransferTx;
+    // @ts-expect-error this wrong but at least now it's clear to us that it is
```

Comment:
these ones would be straightforward to fix the mocks if we wanted to

Reply:
https://github.com/shapeshift/hdwallet/pull/681/commits/5bb93bb034193ccfa4e6cdcd7fadf78a0b0c4a2d